smartsim/_core/_cli/cli.py

```diff
@@ -28,70 +28,84 @@
 
 import argparse
 import sys
-
-from smartsim._core._cli.build import Build
-from smartsim._core._cli.clean import Clean
-from smartsim._core._cli.utils import get_install_path
-
-
-def _usage() -> str:
-    usage = [
-        "smart <command> [<args>]\n",
-        "Commands:",
-        "\tbuild       Build SmartSim dependencies (Redis, RedisAI, ML runtimes)",
-        "\tclean       Remove previous ML runtime installation",
-        "\tclobber     Remove all previous dependency installations",
-        "\nDeveloper:",
-        "\tsite        Print the installation site of SmartSim",
-        "\tdbcli       Print the path to the redis-cli binary" "\n\n",
-    ]
-    return "\n".join(usage)
-
+import typing as t
+
+# from pkg_resources import require
+
+# import smartsim._core._cli as cli
+from smartsim._core._cli.utils import MenuItemConfig
+from smartsim._core._cli.clean import (
+    configure_parser as clean_parser,
+    execute as clean_execute,
+    execute_all as clobber_execute,
+)
+import smartsim._core._cli.clean as clean
```

Comment:
Is `clean` ever used? Looks like you should be free to rm this import

Reply:
great eye. this helps me get rid of my `cleanx` rename, too~